tests/test_temporal_matching.py

```diff
@@ -32,67 +31,40 @@
 '''
 
 
-import pytesmo.temporal_matching as tmatching
-import pandas as pd
+import pytz
 from datetime import datetime
+
+import pandas as pd
 import numpy as np
 import numpy.testing as nptest
 
+import pytesmo.temporal_matching as tmatching
+
 
 def test_df_match_borders():
     """
     Border values can be problematic for temporal matching.
 
     See issue #51
     """
+    ref_df = pd.DataFrame(
+        {"data": np.arange(5)}, index=pd.date_range(
+            datetime(2007, 1, 1, 0),
+            "2007-01-05", freq="D"))
 
-    ref_df = pd.DataFrame({"data": np.arange(5)}, index=pd.date_range(datetime(2007, 1, 1, 0),
-                                                                      "2007-01-05", freq="D"))
     match_df = pd.DataFrame({"matched_data": np.arange(5)},
                             index=[datetime(2007, 1, 1, 9),
                                    datetime(2007, 1, 2, 9),
                                    datetime(2007, 1, 3, 9),
                                    datetime(2007, 1, 4, 9),
                                    datetime(2007, 1, 5, 9)])
-    matched = tmatching.df_match(ref_df, match_df)
-
-    nptest.assert_allclose(
-        np.array([0.375, 0.375, 0.375, 0.375, 0.375]), matched.distance.values)
-    nptest.assert_allclose(np.arange(5), matched.matched_data)
-
 
-def test_df_match_match_on_window_border():
```

Comment:
This test was removed because of the removal for the `asym_window` options?

Since there is no functional replacement for this option with the new parameters the [cci code](https://git.eodc.eu/cci-sm-work/cci_sm_ecvps_py_src/-/blob/master/smecv/steps/temporal_resampling.py#L85-96) will need to be adapted or we need to add the option back.
@tracyscanlon